### PR TITLE
Fix: change color according to theme of web

### PR DESCRIPTION
**Pull request by grgprarup**
- The slide will adapt according to ownCloud web dark and light theme

Fixes https://github.com/JankariTech/web-app-presentation-viewer/issues/17

[Screencast from 2024-02-20 16-22-47.webm](https://github.com/JankariTech/web-app-presentation-viewer/assets/8559121/f4a7603d-d22b-491b-bff6-5099cd555bcc)

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -21,6 +24,9 @@ import 'reveal.js/dist/theme/white.css'
 
 const dataSeparator = '\r?\n---\r?\n'
 const dataSeparatorVertical = '\r?\n--\r?\n'
+const themeStore = useThemeStore()
+let isDarkMode = ref(themeStore.currentTheme.isDark)
```
Comment (saw-jan) [inline src/App.vue:28]:
```suggestion
const isDarkMode = ref(themeStore.currentTheme.isDark)
```

File: src/App.vue [hunk 2]
```diff
@@ -21,6 +24,9 @@ import 'reveal.js/dist/theme/white.css'
 
 const dataSeparator = '\r?\n---\r?\n'
 const dataSeparatorVertical = '\r?\n--\r?\n'
+const themeStore = useThemeStore()
+const isDarkMode = ref(themeStore.currentTheme.isDark)
```
Comment (saw-jan) [inline src/App.vue:28]: if that works well with watch
```suggestion
const { currentTheme } = useThemeStore()
const isDarkMode = ref(currentTheme.isDark)
```

Comment (grgprarup) [reply src/App.vue:28]: That's not working. The class `dark-mode` is not updating with the watch.

**Review by PrajwolAmatya — COMMENTED** [review]
LGTM :+1:

**Review by saw-jan — APPROVED** [review]

**Pull request merged**